modules/rand/internal/types/expected_keepers.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authexported "github.com/cosmos/cosmos-sdk/x/auth/exported"
```

Comment:
use "auth" instead of "authexported"

Reply:
I think the module name should represent the corresponding types package (or alias).

Reply:
yes, u are right